its/core-it-suite/src/test/resources/mng-8385/spi-extension/src/main/resources/META-INF/maven/org.apache.maven.api.di.Inject

```diff
@@ -0,0 +1 @@
+org.apache.maven.its.mng8385.CustomPropertyContributor
```

Comment:
That one should be automatically generated if you compile with enabled annotation processors.
See https://github.com/apache/maven/blob/master/api/maven-api-di/src/main/java/org/apache/maven/di/tool/DiIndexProcessor.java